Modules/_elementtree.c

```diff
@@ -3894,6 +3894,34 @@ _elementtree_XMLParser_close_impl(XMLParserObject *self)
     }
 }
 
+/*[clinic input]
+_elementtree.XMLParser.flush
+
+[clinic start generated code]*/
+
+static PyObject *
+_elementtree_XMLParser_flush_impl(XMLParserObject *self)
+/*[clinic end generated code: output=42fdb8795ca24509 input=effbecdb28715949]*/
+{
+    if (!_check_xmlparser(self)) {
+        return NULL;
+    }
+
+    elementtreestate * const st = self->state;
+
+    if (EXPAT(st, SetReparseDeferralEnabled) == NULL) {
+        return NULL;
+    }
+
+    EXPAT(self->state, SetReparseDeferralEnabled)(self->parser, XML_FALSE);
```

Comment:
Why `self->state` instead of `st`?

Reply:
@serhiy-storchaka that's a leftover, will be gone with the next push in a second.